### ISSUE TITLE
Faster response

### ISSUE THREAD

**Issue by amiralizadeh1** [maziarraissi/PINNs#44]
How can I get a faster result from the code? I don't want anything accurate so 200000 iterations on the Navier-Stokes PDE is too much for me. I changed it to 10000, but the code keeps on running without showing the iteration count, which is strange.

**Comment by amiralizadeh1** [comment]
this is the same with KdV! when the iteration count finishes, the code still runs (showing the loss value only) without printing the final results. there is a savefig() method at the end but it never leaves the training loop to reach it ...

**Comment by Liqian0506** [comment]
you need to change the max iteritation of the LBFGSB optimizer
